portfolio/src/main/webapp/script.js

```diff
@@ -133,4 +134,28 @@ function isElementInViewport(el) {
         rect.bottom <= (window.innerHeight || document.documentElement.clientHeight))
     );
 }
-  
\ No newline at end of file
+
+function checkLogin() {
+    fetch('/check-login')
+        .then(response => response.json())
+        .then((data) => {
+            console.log(data);
+            let successOrFailure = data[0];
+            if(successOrFailure === "true") {
+                console.log("inside if");
```

Comment:
Please remove console logs and system prints

Reply:
done